administration/agent.rst

```diff
@@ -165,10 +130,17 @@ You need to enable the ``asgard2-agent-service`` slider:
    :scale: 40
    :alt: macOS 13 Full Disk Access
 
+If you need to grant Full Disk Access via MDM, please take a look at this appendix:|br|
+:ref:`appendix/mdm-fulldiskaccess:Full Disk Access for macOS asgard2-agent-service via MDM`
+
 .. note:: 
    There is no workaround to this step, since it is an integral
    part of the security design of Apple devices. If you are having trouble
    with THOR scans via ASGARD on macOS, please check if the ``Full Disk
    Access`` permission for the ASGARD agent was granted. Since macOS Mojave
    (v10.14), you need to grant the same permissions to removable volumes,
-   if you plan on scanning those.
\ No newline at end of file
+   if you plan on scanning those.
+
+.. |br| raw:: html
+
+      </br>
```

Comment:
```suggestion
```

We dont need the raw html since we try to avoid using it.

Reply:
This was the only way I found to put the breaks where I want to have them ...